web-registry/src/hooks/useMsgClient.ts

```diff
@@ -11,9 +11,15 @@ interface TxData {
   memo?: string;
 }
 
+interface OptionalCallbacks {
```

Comment:
the changes in this file are potentially the most disruptive, if not implemented correctly.